WcaOnRails/db/migrate/20190216102110_create_regional_organizations.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateRegionalOrganizations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :regional_organizations do |t|
+      t.string :name, null: false
+      t.string :country, null: false
+      t.string :website, null: false
+      t.boolean :currently_acknowledged, default: false, null: false
```

Comment:
I think at this point it's better to involve @thewca/board to ask if we want to keep a history of when organizations were acknowledged? I can then update this as necessary.

Reply:
Keep the date that an organization became acknowledged and or no longer acknowledged? I see no harm in keeping it. We already store the former in our spreadsheet that compiles the results of the applications, but it would be better to keep it on the website, I think.

Reply:
I have added `start_date` and `end_date` fields instead of the `currently_acknowledged` one!